misc/network.c

```diff
@@ -1220,8 +1223,10 @@ read_stream_connection_unbuffered (int fd, void *buf0, int min_len, int max_len)
 
     default:
       if (fp->transport || fp->fd != 0)
-        g_message ("Severe bug! Unhandled transport layer %d (fd=%d)",
-                   fp->transport, fd);
+        g_message ("Severe bug! Unhandled transport layer %d (fd=%d). "
+                   "Function %s called from %s.",
+                   fp->transport, fd, nasl_get_plugin_filename (),
+                   nasl_get_function_name ());
```

Comment:
nasl_get_plugin_filename () and nasl_get_function_name () seems to have the wrong order